src/app/pages/ElectronicDelivery.jsx

```diff
@@ -236,30 +231,33 @@ class ElectronicDelivery extends React.Component {
               {title}
             </Link>
           </h2>
-          {
-            callNo && (
-              <div className="call-number">
-                <span>Call Number:</span><br />
-                {callNo}
-              </div>
-            )
-          }
+          {callNo && (
+            <div className='call-number'>
+              <span>Call Number:</span>
+              <br />
+              {callNo}
+            </div>
+          )}
         </div>
-
-        <div>
-          {
-            !_isEmpty(raiseError) && (
-              <div className="nypl-form-error" ref="nypl-form-error">
+        {!eddRequestable ? (
```

Comment:
@danamansana points out that this may not yet have the support it needs from the production DiscoveryAPI